doc/cephadm/host-management.rst

```diff
@@ -316,10 +316,32 @@ create a new CRUSH host located in the specified hierarchy.
 
   The ``location`` attribute will be only affect the initial CRUSH location. Subsequent
   changes of the ``location`` property will be ignored. Also, removing a host will not remove
-  any CRUSH buckets.
+  any CRUSH buckets unless the ``--rm-crush-entry`` flag is provided to the ``orch host rm`` command
 
 See also :ref:`crush_map_default_types`.
 
+Removing a host from the CRUSH map
+==================================
+
+The ``ceph orch host rm`` command has support for removing the bucket entry for the host
+in the CRUSH map. This is done by providing the ``--rm-crush-entry`` flag.
+
+.. prompt:: bash [ceph:root@host1/]#
+
+   ceph orch host rm host1 --rm-crush-entry
+
+When this flag is specified, cephadm will make a best-effort attempt to remove the bucket
+entry for the host from the CRUSH map. Note that if it fails to do so, the host will still
+have been removed from cephadm's management, but its entry in the CRUSH map will remain.
```

Comment:
wondering if we should implement a health check for when this fails?

Reply:
Now that this doesn't remove the host from the inventory if removal from the crush map fails as per your other comment, I assume this is no longer necessary.